plugins/janus_streaming.c

```diff
@@ -118,12 +118,13 @@ rtspiface = network interface IP address or device name to listen on when receiv
  * (invalid JSON, invalid request) which will always result in a
  * synchronous error response even for asynchronous requests.
  *
- * \c list , \c create , \c destroy , \c recording , \c enable and
+ * \c list , \c create , \c destroy , \c recording , \c edit , \c enable and
  * \c disable are synchronous requests, which means you'll
  * get a response directly within the context of the transaction. \c list
  * lists all the available streams; \c create allows you to create a new
  * mountpoint dynamically, as an alternative to using the configuration
- * file; \c destroy removes a mountpoint and destroys it; \c recording
+ * file; + * dynamically, as an alternative to using the configuration file; 
+ * \c edit allows you to dynamically edit some mountpoint properties (e.g., the PIN);
```

Comment:
Is the line before this a leftover/typo?

Reply:
Yea, missed a newline!

Reply:
There's an extra `+ *` too, though.